syntax = "proto3";
package nlp;

message GenerateAllRequest {
    uint32 userId = 1;
}

//client streaming
message GenerateAllResponse{
    bool start = 1;
}


service NlpService {
    rpc GenerateAll(GenerateAllRequest) returns (GenerateAllResponse) {}
}
